### ISSUE TITLE
gem install fpm not work

### ISSUE THREAD

**Issue by jjaing** [jordansissel/fpm#926]
ERROR:  Could not find a valid gem 'fpm' (>= 0), here is why:
          Unable to download data from https://rubygems.org/ - Errno::ECONNRESET: Connection reset by peer - SSL_connect (https://rubygems.org/quick/Marshal.4.8/fpm-1.3.3.gemspec.rz)

**Comment by hatt** [comment]
This is just the gem command telling you it had a network connectivity problem and has nothing to do with fpm itself. Just wait a bit and give it another try, for whatever reason your machine couldn't reach rubygems.org at the time.

**Comment by jjaing** [comment]
Yes, this is not fpm's problem. I can reach to rubygems.org, but the URL for fpm 1.3.3 is unavailable, I believe this is not my connection's problem. I am not sure where to post this issue. Maybe not here. sorry!

**Comment by hatt** [comment]
Rubygems.org will redirect to an actual CDN but that looks like the webserver just dropped your connection. The 1.3.3 gem is definitely there (I got 302'd to https://rubygems.global.ssl.fastly.net/quick/Marshal.4.8/fpm-1.3.3.gemspec.rz when I tried it) but it might be a bad mirror or the server acting up. Did you try just once, or multiple times on different networks? I think this is a bug for the rubygems.org team more than FPM, but if we can resolve it here then we may as well.

**Comment by jjaing** [comment]
Thanks for your response! I just tried multiple times on same network. I don't have alternative network.

**Comment by darkshade9** [comment]
I am also seeing this problem.

$ gem install fpm
WARNING:  RubyGems 1.2+ index not found for:


RubyGems will revert to legacy indexes degrading performance.
ERROR:  Could not find a valid gem 'fpm' (>= 0) in any repository

**Comment by don-rumata** [comment]
For Debain\Ubuntu: `sudo apt -y install ruby ruby-dev rubygems build-essential`
